aptos-move/framework/table-natives/src/lib.rs

```diff
@@ -129,11 +141,17 @@ impl<'a> NativeTableContext<'a> {
                 match op {
                     Op::New(val) => {
                         let bytes = serialize(&value_layout_info.layout, &val)?;
-                        entries.insert(key, Op::New(bytes.into()));
+                        let layout = value_layout_info
+                            .has_aggregator_lifting
+                            .then(|| value_layout_info.layout.clone());
```

Comment:
I think layout is owned, no need to clone?

Reply:
Tried it now. It then gives the borrow of moved value error.

Reply:
hm, can you paste it? where is it borrowed after move?